migrator/zenodo_rdm_migrator/transform/entries/records/records.py

```diff
@@ -14,7 +14,7 @@
 from .custom_fields import ZenodoCustomFieldsEntry
 from .metadata import ZenodoDraftMetadataEntry, ZenodoRecordMetadataEntry
 
-ZENODO_DATACITE_PREFIX = "10.5281"
+ZENODO_DATACITE_PREFIX = "10.5281/"
```

Comment:
where is this used? https://github.com/search?q=repo%3Azenodo%2Fzenodo-rdm%20DATACITE_PREFIX&type=code